runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -1704,6 +1704,13 @@
 	 */
 	void setCompass(Sprite Sprite);
 
+	/**
+	 * Sets whether inventory quantity is verbose.
+	 *
+	 * @param state verbose state
+	 */
+	void setVerbose(boolean state);
```

Comment:
Change this method name to actually describe what's made verbose (ie setItemQuantitiesVerbose)